light-client/src/components/verifier.rs

```diff
@@ -15,7 +15,7 @@ use serde::{Deserialize, Serialize};
 
 /// Represents the result of the verification performed by the
 /// verifier component.
-#[derive(Debug, Serialize, Deserialize)]
+#[derive(Debug, Serialize, Deserialize, PartialEq)]
```

Comment:
Can we also add `Eq` here while we're at it?

Reply:
Done!